frontend/javascripts/oxalis/model/accessors/dataset_accessor.ts

```diff
@@ -827,3 +827,11 @@ export function getEffectiveIntensityRange(
 
   return layerConfiguration.intensityRange || defaultIntensityRange;
 }
+
+// Note that this prop is lazy-loaded. Call an ensureSegmentIndexIsLoadedAction to make sure this info was fetched.
+export function getMaybeSegmentIndexAvailability(
+  dataset: APIDataset,
+  layerName: string | undefined,
```

Comment:
I find it a bit weird that this function accepts undefined and then uses it to compare it with the layerName. I'd recommend to do one of the two options:
- don't accept undefined and adapt the caller to avoid this case (since react hooks cannot be called within ifs, you would need to do something like this: ¹)
- add a `if (visibleSegmentationLayer == null) { return false }` at the top of this function here

¹:
```
useSelector((state: OxalisState) =>
    visibleSegmentationLayer != null ? getMaybeSegmentIndexAvailability(state.dataset, visibleSegmentationLayer.name) : false,
  );
```